### DISCUSSION TITLE
Facebook comments reply via Chatwoot

### DISCUSSION THREAD

**Discussion by harismateen** [chatwoot/chatwoot#8455] [Feature Requests]
Hello,

Can integrate Facebook post comments also into chatwoot?

**Comment by joseph0063** [comment]
Hey, any news on this feature request ?
It would be a very useful functionality, I think all other solutions integrate it.
Thank you guys for the amazing work.

**Comment by pranavrajs** [comment]
Moving this to feature discussions as we won't be prioritizing this soon.

**Comment by sojan-official** [comment]
Hello Chatwoot Users,

Our current mission at Chatwoot is centered on refining customer support and fostering better direct customer communication. We recognize the interest in integrating social media comment management. However, this introduces a significant challenge due to our existing UI, which might not be fully equipped to support this new feature effectively.

As an interim measure, we suggest utilizing another tool specifically designed for managing social media comments which can help by automatically generating responses that guide users towards direct messages (DMs), a process Chatwoot is adept at handling.

Your feedback is invaluable to us, and it plays a crucial role in our development process. We are open to the idea of adding comment management capabilities in the future, with user demand and feedback guiding our decisions.

Thank you for your understanding and for being an integral part of our journey.

Warm regards,
The Chatwoot Team

**Reply by joseph0063** [reply]
thank you for your answer appreciate your time and dedication

**Comment by karimsiam** [comment]
Hello,

I would suggest that a every comment to be handled as a separate conversation with all the replies related to that comment so in that way it can be utilized within the same UI we are having today within chatwoot.

**Comment by tanpham380** [comment]
I think this future need for this project, because agent can repiles comment like chatting.

**Comment by ehstbr** [comment]
This functionality is very important not only for marketing, but for customer service as well. Complaints and demands often arrive in the comments of posts on Facebook or Instagram Reels. I believe that the contact on ChatWoot should be the author of the comment and if other users also comment, just use the name with a link to click and see that person's profile.

I thought about creating an integration via WebHook, but native integration would be the most appropriate in my opinion.

**Comment by nacholupotti** [comment]
Hello feedback here, we do AI agents and there is more and more customers that are asking about answering with AI both Instagram and Facebook comments

**Comment by md-riaz** [comment]
Meta is merging inbox messages and post comments also as inbox message with just extra note

**Comment by md-riaz** [comment]
https://github.com/chatwoot/chatwoot/commit/4b2c1bdaecb33da452d5113c40ba0f701f76d056

**Comment by md-riaz** [comment]
<img width="722" height="931" alt="image" src="https://github.com/user-attachments/assets/0ab782b4-6495-4630-861a-e3f71efa8184" />
ui can be like this
